docs/1-simple-conversation-objc.md

```diff
@@ -0,0 +1,282 @@
+            
+            NSLog(@"DEMO - chat self.client.account.user is nil");
+            
+            return;
+        }
+        
+        NSLog(@"DEMO - chat creation unsuccessful with %@", error.localizedDescription);
+        
+    } onComplete:^{
+        //Execute code upon completion
+    }];
+    
+    [self performSegueWithIdentifier:@"chatView" sender: nil];
+}
+```
+
+### 2.5 Navigate to ChatViewController
+
+As we mentioned above, creating a conversation results from a call to the the new() method. In the absence of a server we’ll ‘simulate’ the creation of a conversation within the app when the user clicks the chatBtn.
+
+When we construct the segue for `ChatViewController`, we pass the first conversation so that the new controller. Remember that the `CONVERSATION_ID` comes from the id generated in [step 1.3](#13---create-a-conversation).
+
+```Objective-C 
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    
+    ChatVC *chatVC = segue.destinationViewController;
+    chatVC.conversation = client.conversation.conversations.conversationsObjc.firstObject;
+    //    NXMConversationCollection *conversations = client.conversation.conversations;
+    //    chatVC.conversation = [conversations indexWithSafe: conversations.startIndex];
+}
+```
+
+### 2.6 Create the Chat layout
+
+
+Configure your own layout for chat with: 
+
+1. an instance of UITextView called `textView`
+2. an instance of UITextField called `textField`
+3. an instance of UIButton called `sendBtn`
+
+
+### 2.7 Create the ChatVC
+
+Like last time we'll wire up the views in ChatVC. We also need to grab the reference to `conversation` from the incoming view controller.
+
+```Objective-C
+#import <UIKit/UIKit.h>
+@import Stitch;
+
+@interface ChatVC : UIViewController
+
+@property (retain, nonatomic) NXMConversation *conversation;
+
+@property (weak, nonatomic) IBOutlet UITextView *textView;
+
+@property (weak, nonatomic) IBOutlet UITextField *textField;
+
+@end
+```
+
+### 2.8 - Sending `text` Events
+
+To send a message we simply need to call `send()` on our instance of `conversation`. `send()` takes one argument, a `String message`.
+
+```Objective-C
+- (IBAction)sendBtn:(id)sender {
+    
+    NSString *text = _textField.text ;
+    
+    if (text.length > 0) {
+        [conversation sendText: text error: NULL];
+    }
+}
+```
+
+### 2.9 - Receiving `text` Events
+
+In `viewDidLoad()` we want to add a handler for handling new events like the TextEvents we create when we press the send button. We can do this like so:
+
+```Objective-C 
+NXMEventCollection *events = [conversation events];
+    
+NXMEventBase *baseEvent = [events indexWithSafe:[events startIndex]];
+    
+[baseEvent isCurrentlyBeingSent];
+```
+
+## 3.0 - Trying it out
+
+After this you should be able to run the app and send messages.
```

Comment:
add a link to the code example folder on GitHub please

Reply:
Added.